docs/themes/getting_started.rst

```diff
@@ -1,4 +1,182 @@
 Getting Started
 ==========================================================
 
+Themes can be used to create default content and layouts for emails and landing pages. Content is written in the `Twig templating language <https://twig.symfony.com/>`__.
 
+They can also be used to customize form pages (when visiting /form/ID) or using an iframe to embed a form into a 3rd party page. One can also customize the form structure and fields but this must be done with PHP files which disqualifies the theme from being installable through Mautic's Theme manager and manually uploaded to the server.
+
+Finally, a special ``system`` theme can be used to override system templates to avoid customizing core code that could be lost with future upgrades.
+
+Directory Structure
+==========================================================
+
+A theme is made of three components: the config, twig files (HTML content), and thumbnails::
+
+    config.json
+        html/
+            email.html.twig
+            form.html.twig
+            message.html.twig
+            page.html.twig
+    thumbnail.png
+    thumbnail_email.png
+    thumbnail_form.png
+    thumbnail_page.png
+
+Config File
+-----------
+
+The config file tells Mautic how to utilize the theme.
+
+.. code-block:: json
+
+    {
+      "name": "Theme Name",
+      "author": "John Doe",
+      "authorUrl": "https://john-doe-the-mautic-theme-builder.com",
+      "features": [
+        "email",
+        "form",
+        "page"
+      ],
+      "builder": ["legacy", "grapesjsbuilder"]
+    }
+
+name
+    This is the name of the theme as displayed in Mautic.
+author
+    This displays in the theme manager as credit to the author of the theme.
+authorUrl
+    URL used for the author link.
+features
+    An array of strings that tells Mautic which features the theme supports. Currently recognized values are ``email`` (email builder template), ``form`` (formatting the form page) and ``page`` (page builder template). A corresponding ``html/[feature].html.twig`` file is required for each feature supported. For example, if the theme supports ``email``, then there should be a ``html/email.html.twig`` file. See Twig Files more information on each feature.
+builder
+    Array of strings for which builder the theme supports. Currently only applies to themes that support ``page`` or ``email``. By default, themes without this will only be recognized by Mautic's legacy builder. New themes built should declare the specific builders it supports.
+
+Twig Files
+-------------------------
+
+html/message.html.twig
+^^^^^^^^^^^^^^^^^^^^^^
+
+This file is a simple message file mainly used as the landing page for when a lead unsubscribes or resubscribes to the system’s emails. But may be used by other areas so should be included in all themes.
```

Comment:
```suggestion
This file is mainly used as the Landing Page for when a Lead unsubscribes or resubscribes to the system’s emails. It may be used by other areas, so should be included in all Themes.
```